src/lib/Components/Inbox/Conversations/index.tsx

```diff
@@ -99,7 +101,7 @@ export class Conversations extends React.Component<Props, State> {
         })
 
         if (!this.props.me.conversations.pageInfo.hasNextPage) {
-          this.setState({ completed: true })
+          this.setState({ completed: true, initialLoadDone: true })
```

Comment:
Isn't it possible for there to be more pages after the initial load? i.e. is `initialLoadDone` only being set to `true` when `hasNextPage` is `false`?